visualization/app/codeCharta/ui/fileChooser/fileChooser.component.ts

```diff
@@ -52,18 +52,38 @@ export class FileChooserController {
 					}
 
 					if (readFiles === element.files.length) {
-						this.setNewData()
+						this.setNewData(element.files)
 					}
 				}
 			}
 		})
 	}
 
-	setNewData() {
+	setNewData(sortingArray ?) {
+		if (sortingArray!== undefined) {
```

Comment:
Please run the prettier/formatter once :)

Reply:
I always forget abouth that :)